Cabal/Distribution/Simple/Program/GHC.hs

```diff
@@ -172,11 +172,15 @@ normaliseGhcArgs (Just ghcVersion) PackageDescription{..} ghcArgs
               , "break-on-exception", "print-bind-result"
               , "print-bind-contents", "print-evld-with-show"
               , "implicit-import-qualified", "error-spans"
+              , "print-expanded-synonyms", "print-potential-instances"
```

Comment:
It seems these should be under some specific GHC version? I went through all the flags in the GHC source at the time this code was written and since I didn't see them then I assume these are new in ghc 8.6/8.8? Other than that this seems fine.

Reply:
I did this just now and they're already in 8.0

```
sergey@debian:~$ ghc --version
The Glorious Glasgow Haskell Compilation System, version 8.0.2
sergey@debian:~$ ghc --show-options | awk '/-fprint/'
-fprint-explicit-foralls
-fprint-explicit-kinds
-fprint-explicit-coercions
-fprint-explicit-runtime-reps
-fprint-equality-relations
-fprint-unicode-syntax
-fprint-expanded-synonyms
-fprint-potential-instances
-fprint-typechecker-elaboration
```

Which means that my placement of `print-unicode-syntax` is not entirely correct and `-fprint-explicit-runtime-reps` seems to be missing. Perhaps the better way is to treat all flags starting with `-fprint` as simple and safe to ignore?